### ISSUE TITLE
Getting nan as loss when training without modifications

### ISSUE THREAD

**Issue by richardlyf** [nianticlabs/monodepth2#317]
Hi, I'm trying to run the training as is without modifications, but for some reason, I'm getting nans.
I cloned the repo and downloaded KITTI using the commands specified in the readme. I'm running training on a google cloud VM with NVIDIA Tesla P100 GPU. 
The command I'm using is: `python train.py --model_name mono_model --data_path /home/jupyter/kitti/ --batch_size 6`, and I'm getting this as the result:
![training](https://user-images.githubusercontent.com/23286980/110071651-40dc9700-7d31-11eb-9a40-c419950ed158.PNG)
Why could this be happening? Thank you for your help!

**Comment by mrharicot** [comment]
Hi,
You don't seem to be using pytorch 0.4.1 as specified in the setup:
https://github.com/nianticlabs/monodepth2#%EF%B8%8F-setup

**Comment by richardlyf** [comment]
Ah yes, that's exactly the issue! I'm surprised that it ran at all. Thank you!
